server/sso/session_store.go

```diff
@@ -13,6 +13,8 @@ import (
 	redigo "github.com/gomodule/redigo/redis"
 )
 
+var ErrSessionNotFound = fleet.NewAuthRequiredError("session not found")
```

Comment:
To have a unique error UUID we should not define this globally and just return a new error every time:
```sh
return fleet.NewAuthRequiredError("session not found")
```

Reply:
@lucasmrod thanks! could you give this another look?